documentation/content/en/books/handbook/basics/_index.adoc

```diff
@@ -717,7 +717,7 @@ When reading the "Directory Listing" column, a `-` is used to represent a permis
 |`rwx`
 |===
 
-Pass the `-l` argument to man:ls[1] to view a long directory listing that includes a column of information about a file's permissions for the owner, group, and everyone else.
+Use the `-l` argument when invoking man:ls[1] to view a long directory listing that includes a column of information about a file's permissions for the owner, group, and everyone else.
```

Comment:
That works but feels a bit stilted. How about "use... with..."?

Reply:
Much better, thanks.